vault/logical_system.go

```diff
@@ -1665,13 +1665,22 @@ func (b *SystemBackend) handleMount(ctx context.Context, req *logical.Request, d
 		config.PassthroughRequestHeaders = apiConfig.PassthroughRequestHeaders
 	}
 
-	// Alias versioned KV
-	if logicalType == "vkv" {
+	// Alias KV v1
+	if logicalType == "kv-v1" {
```

Comment:
Can these alias checks be moved into the switch in https://github.com/hashicorp/vault/pull/4293/files#diff-a145099c8e3d917858619dcfb2ae09b9L1627?